app/src/main/java/org/gdg/frisbee/android/eventseries/TaggedEventSeries.java

```diff
@@ -18,22 +17,22 @@
     private long mSartDateInMillis;
     private long mEndDateInMillis;
 
-
-    public TaggedEvent(String tag, int drawerIconResId, int titleResId, int descriptionResId, int logoResId, long endDateInMillis) {
+    public TaggedEventSeries(String tag, 
+                             @DrawableRes int drawerIconResId, 
+                             @StringRes int titleResId, 
+                             @StringRes int descriptionResId, 
+                             @DrawableRes int logoResId,
+                             long startDateInMillis,
+                             long endDateInMillis) {
         mTag = tag;
         mDrawerIconResId = drawerIconResId;
         mTitleResId = titleResId;
         mDescriptionResId = descriptionResId;
         mLogoResId = logoResId;
-        mSartDateInMillis = DateTime.now().getMillis();
+        mSartDateInMillis = startDateInMillis;
```

Comment:
Just a small typo: `mSartDateInMillis` should be renamed to `mStartDateInMillis`.